src/osd/OSDMap.cc

```diff
@@ -5157,6 +5157,21 @@ void OSDMap::check_health(health_check_map_t *checks) const
     }
   }
 
+  auto& d = checks->add("POOL_SCRUB_FLAGS", HEALTH_OK, "Some pool(s) have the noscrub, nodeep-scrub flag(s) set");
```

Comment:
pretty sure this needs to be wrapped.

Reply:
also, would be nice if we have the message providing whether it's just `noscrub` or `nodeep-scrub`, or both.

Reply:
Oops, of course it needs to wrapped. Fixed. Per your second comment, I made it a bit better, but we can't really know if pools have one flag, or the other, or both. I think it's acceptable though as one can easily find it using `ceph health detail`.